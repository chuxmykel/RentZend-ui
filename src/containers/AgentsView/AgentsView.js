import React from 'react';
import AgentCard from '../../components/AgentCard';
import data from '../../utils/dummy-data';
import './AgentsView.css';

const AgentsView = () => {
  return (
    <div className="agents-view">
      {data.map(({ id, name, email, phoneNumber, address, zipCode, passport }) => (
        <AgentCard
          key={id}
          name={name}
          email={email}
          phoneNumber={phoneNumber}
          address={address}
          zipCode={zipCode}
          passport={passport}
        />
      ))}
    </div>
  );
};

export default AgentsView;
