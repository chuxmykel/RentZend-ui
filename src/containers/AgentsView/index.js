import AgentsView from './AgentsView';

export default AgentsView;
