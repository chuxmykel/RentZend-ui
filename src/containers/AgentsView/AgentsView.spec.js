import React from 'react';
import { shallow } from 'enzyme';
import AgentsView from './AgentsView';

describe('<AgentsView /> with no props', () => {
  const container = shallow(<AgentsView />);
  it('should match the snapshot', () => {
    expect(container.html()).toMatchSnapshot();
  });

  it('should have the main div', () => {
    expect(container.find('div').length).toEqual(1);
  });
});
