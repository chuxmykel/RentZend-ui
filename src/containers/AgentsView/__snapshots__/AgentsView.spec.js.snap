// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AgentsView /> with no props should match the snapshot 1`] = `"<div class=\\"agents-view\\"><div class=\\"MuiPaper-root MuiCard-root agent-card MuiPaper-elevation8 MuiPaper-rounded\\"><div class=\\"MuiCardContent-root agent-card-content\\"><div class=\\"MuiAvatar-root MuiAvatar-circle makeStyles-large-1\\"><img alt=\\"agent - James Okorocha\\" src=\\"https://randomuser.me/api/portraits/men/75.jpg\\" class=\\"MuiAvatar-img\\"/></div><h4 class=\\"MuiTypography-root MuiTypography-h4\\">James Okorocha</h4><h6 class=\\"MuiTypography-root MuiTypography-subtitle1\\">james.o@gmail.com</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">(123) 456-7899</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">no. 10 Ojoro Street. Nakupenda</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">zip code: 101233</h6></div></div><div class=\\"MuiPaper-root MuiCard-root agent-card MuiPaper-elevation8 MuiPaper-rounded\\"><div class=\\"MuiCardContent-root agent-card-content\\"><div class=\\"MuiAvatar-root MuiAvatar-circle makeStyles-large-1\\"><img alt=\\"agent - Mitesh Kelji\\" src=\\"https://randomuser.me/api/portraits/men/43.jpg\\" class=\\"MuiAvatar-img\\"/></div><h4 class=\\"MuiTypography-root MuiTypography-h4\\">Mitesh Kelji</h4><h6 class=\\"MuiTypography-root MuiTypography-subtitle1\\">m.kelji.@gmail.com</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">(098) 765-4321</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">no.15, Maracana Arena, Andra Pradesh</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">zip code: 101653</h6></div></div><div class=\\"MuiPaper-root MuiCard-root agent-card MuiPaper-elevation8 MuiPaper-rounded\\"><div class=\\"MuiCardContent-root agent-card-content\\"><div class=\\"MuiAvatar-root MuiAvatar-circle makeStyles-large-1\\"><img alt=\\"agent - Moses Obika\\" src=\\"https://randomuser.me/api/portraits/men/28.jpg\\" class=\\"MuiAvatar-img\\"/></div><h4 class=\\"MuiTypography-root MuiTypography-h4\\">Moses Obika</h4><h6 class=\\"MuiTypography-root MuiTypography-subtitle1\\">obika.moses.@gmail.com</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">(132) 758-9453</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">9278 New road, Kilcoole, waterford</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">zip code: 93027</h6></div></div><div class=\\"MuiPaper-root MuiCard-root agent-card MuiPaper-elevation8 MuiPaper-rounded\\"><div class=\\"MuiCardContent-root agent-card-content\\"><div class=\\"MuiAvatar-root MuiAvatar-circle makeStyles-large-1\\"><img alt=\\"agent - Ikechukwu Okey\\" src=\\"https://randomuser.me/api/portraits/men/22.jpg\\" class=\\"MuiAvatar-img\\"/></div><h4 class=\\"MuiTypography-root MuiTypography-h4\\">Ikechukwu Okey</h4><h6 class=\\"MuiTypography-root MuiTypography-subtitle1\\">okey.ikechukwu.@gmail.com</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">(234) 928-4253</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">53 Haven lane, Lagos Island</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">zip code: 101254</h6></div></div><div class=\\"MuiPaper-root MuiCard-root agent-card MuiPaper-elevation8 MuiPaper-rounded\\"><div class=\\"MuiCardContent-root agent-card-content\\"><div class=\\"MuiAvatar-root MuiAvatar-circle makeStyles-large-1\\"><img alt=\\"agent - Flavour Umeh\\" src=\\"https://randomuser.me/api/portraits/women/24.jpg\\" class=\\"MuiAvatar-img\\"/></div><h4 class=\\"MuiTypography-root MuiTypography-h4\\">Flavour Umeh</h4><h6 class=\\"MuiTypography-root MuiTypography-subtitle1\\">flavour.baby@gmail.com</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">(928) 674-4390</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">Rwang Pam Stadium, Jos, Plateau State</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">zip code: 101254</h6></div></div><div class=\\"MuiPaper-root MuiCard-root agent-card MuiPaper-elevation8 MuiPaper-rounded\\"><div class=\\"MuiCardContent-root agent-card-content\\"><div class=\\"MuiAvatar-root MuiAvatar-circle makeStyles-large-1\\"><img alt=\\"agent - Kelechi Ngwobia\\" src=\\"https://randomuser.me/api/portraits/women/75.jpg\\" class=\\"MuiAvatar-img\\"/></div><h4 class=\\"MuiTypography-root MuiTypography-h4\\">Kelechi Ngwobia</h4><h6 class=\\"MuiTypography-root MuiTypography-subtitle1\\">kcmykairl@gmail.com</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">(287) 878-2833</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">Old Legislative Qtrs. Angwan Rukuba, Plateau State</h6><h6 class=\\"MuiTypography-root MuiTypography-subtitle2\\">zip code: 101254</h6></div></div></div>"`;
